src/components/NavPanel.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { PropTypes } from 'react';
+import { Nav, NavItem } from 'react-bootstrap';
+import styles from './../../public/stylesheets/nav.css';
+
+
+const NavPanel = ({ active }) => (
+  <div className={styles.navpanel}>
+    <Nav bsStyle="pills" stacked activeKey={active} >
+      <NavItem eventKey={1} className={styles.item} href="/">Home</NavItem>
+      <NavItem eventKey={2} className={styles.item} href="#">Create/Edit Plan of Study</NavItem>
```

Comment:
This element changes color on mouseover, and back when you mouseoff, but if you click before you mouseoff, the color doesn't change back like it should. Probably it'll just redirect you to a different page eventually, because navbar, but the behavior seems unintentional.